AuthenticatorShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -132,3 +132,16 @@
 "MoveToBitwarden" = "Move to Bitwarden";
 "AddCodeToBitwarden" = "Add code to Bitwarden";
 "AddCodeLocally" = "Add code locally";
+"ScanComplete" = "Scan complete";
+"SaveThisAuthenticatorKeyHereOrAddItToALoginInYourBitwardenApp" = "Save this authenticator key here, or add it to a login in your Bitwarden app.";
+"SaveHere" = "Save here";
+"TakeMeToBitwarden" = "Take me to Bitwarden";
+"SetSaveToBitwardenAsYourDefaultSaveOption" = "Set Save to Bitwarden as your default save option?";
+"SetSaveLocallyAsYourDefaultSaveOption" = "Set Save locally as your default save option?";
```

Comment:
🤔 Does it make sense to uppercase "save" here?

Reply:
I'll double check with Emily. I feel like these should have quotes or something - the possible values are "Save to Bitwarden" and "Save locally" (i.e. with caps) but in the middle of a sentence it looks a bit odd.

Reply:
Followed up with Emily she agreed to add quotes around the options. I've added that in the latest.